modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/JavaStagedModelDataHandlerCheck.java

```diff
@@ -101,12 +101,18 @@ protected String doProcess(
 		String existingVariableName = matcher.group(3);
 		String importedVariableName = matcher.group(2);
 
-		if (stagedModelType.equals(
-				getVariableTypeName(
-					javaMethodContent, fileContent, existingVariableName)) &&
-			stagedModelType.equals(
-				getVariableTypeName(
-					javaMethodContent, fileContent, importedVariableName))) {
+		String variableTypeName1 = getVariableTypeName(
+			javaMethodContent, fileContent, fileName, existingVariableName);
+
+		String variableTypeName2 = getVariableTypeName(
+			javaMethodContent, fileContent, fileName, importedVariableName);
```

Comment:
@TamyrisTorres, please rename to `existingVariableTypeName` and `importedVariableTypeName`